README.md

```diff
@@ -6,24 +6,32 @@ Exhausts the keyspace on my Ryzen 3700X in 4 minutes and 11 seconds which equals
 
 ## Installation
 
-On Linux: grab a binary from the [releases-page](https://github.com/MaxMatti/aax-bruteforce/releases) or build it yourself, then run it.
+### Linux
+
+Grab a binary from the [releases-page](https://github.com/MaxMatti/aax-bruteforce/releases) or build it yourself, then run it.
+
+People running Arch Linux(or derivatives) can use the [AUR package](https://aur.archlinux.org/packages/aax-bruteforce).
```

Comment:
That's a 404 for me, do you intend to create this?

Reply:
Yep, working on it right now